primitive/result.go

```diff
@@ -212,8 +214,12 @@ func DecodeMessage(data []byte) []*MessageExt {
 	return msgs
 }
 
-func createMessageId(addr []byte, offset int64) string {
-	return "msgID" // TODO
+func createMessageId(addr []byte, port int32, offset int64) string {
```

Comment:
align Java, remove port

Reply:
java port is inside the addr, but in golang related impl, addr is independent to port.